extension/chrome/elements/compose.ts

```diff
@@ -260,6 +272,57 @@ export class ComposeView extends View {
   };
 
   public isCustomerUrlFesUsed = () => Boolean(this.fesUrl);
+
+  private preParseEmailRecipientsIfNeeded = async () => {
+    if (Catch.isThunderbirdMail() && this.thunderbirdMsgId) {
+      const thunderbirdMimeMsg = await browser.messages.getFull(this.thunderbirdMsgId);
```

Comment:
I think we can replace it with `const { headers, parts }  = await browser.messages.getFull(this.thunderbirdMsgId);`
And then instead of using `thunderbirdMimeMsg.headers...` and `thunderbirdMimeMsg.parts...` we can just type `headers...` and `parts...`, so no need for duplicating `thunderbirdMimeMsg`

Reply:
Thank you - I will update it accordingly.